excel_import_export/models/xlsx_template.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+import os
```

Comment:
Please add copyright in all headers
encoding not necessary as of v11

Reply:
Done